src/components/SystemMetrics.jsx

```diff
@@ -20,24 +33,33 @@ const dbData =
 const transHistory = dbData ? dbData.webappActions : [];
 
 export default function SystemMetrics() {
+  const [tvl, setTvl] = useState(0);
   var volume = 200000
   var details = [
     {
         title: "Total Value Locked (TVL)",
-        val: `${88.3}M ALGO`,
+        val: `$${tvl.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",")}`,
         hasToolTip: true,
     },
     {
-      title: "Volume",
-      val: `$${volume.toLocaleString("en-US")}`,
+      title: "Staked GARD",
+      val: `${0}`,
```

Comment:
Is there any reason this is hard set to 0 other than we haven't released staking as a feature? Will prob need an update function like Tvl later on